### PR TITLE
Cs/pool en route

### PR DESCRIPTION
**Pull request by colinsheppard**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/2013)

### CODE REVIEW

File: src/main/scala/beam/agentsim/agents/MobilityRequest.scala [hunk 1]
```diff
@@ -35,6 +40,39 @@ case class MobilityRequest(
       case Some(p) => p.personId.toString
       case None    => "None"
     }
-    s"${formatTime(time)}|$tag|${personid}|${activity.getType}| => ${formatTime(serviceTime)}"
+//    s"${formatTime(baselineNonPooledTime)}|$tag|${personid}|${activity.getType}| => ${formatTime(serviceTime)}"
+    s"${baselineNonPooledTime}|$tag|${personid}|${activity.getCoord}| => ${serviceTime}"
+  }
+  override def equals(that: Any): Boolean = {
```
Comment (REASY) [inline src/main/scala/beam/agentsim/agents/MobilityRequest.scala]: This is tricky and can cause different issues in the future when you have a collection (List/Vector/Map) of `MobilityRequest` and want to find an instance of `MobilityRequest`.

File: src/main/resources/beam-template.conf [hunk 2]
```diff
@@ -144,7 +144,7 @@ beam.agentsim.agents.rideHail.allocationManager.repositionLowWaitingTimes.waitin
 beam.agentsim.agents.rideHail.allocationManager.repositionLowWaitingTimes.demandWeight = "double | 4.0"
 beam.agentsim.agents.rideHail.allocationManager.repositionLowWaitingTimes.produceDebugImages = true
 beam.agentsim.agents.rideHail.allocationManager.alonsoMora.pickupTimeWindowInSec = "int | 360"
-beam.agentsim.agents.rideHail.allocationManager.alonsoMora.dropoffTimeWindowInSec = "int | 600"
+beam.agentsim.agents.rideHail.allocationManager.alonsoMora.dropoffTimeWindowInSec= "double | 0.2"
```
Comment (wrashid) [inline src/main/resources/beam-template.conf]: this seems too low 0.2 seconds...

Comment (colinsheppard) [reply src/main/resources/beam-template.conf]: It has been changed into a multiplier in a linear model. But it should have a new name.

File: src/main/scala/beam/agentsim/agents/MobilityRequest.scala (hunk 1 above)
Comment (colinsheppard) [reply src/main/scala/beam/agentsim/agents/MobilityRequest.scala]: Agree, but there's an important reason for doing it.

File: src/main/resources/beam-template.conf (hunk 2 above)
Comment (wrashid) [reply src/main/resources/beam-template.conf]: ok, got it

**Pull request merged**